### DISCUSSION TITLE
Status of the project?

### DISCUSSION THREAD

**Discussion by Gavin-Williams** [CommunityToolkit/Windows#554] [Q&A] [answered]
I was just told that CommunityToolkit.WinUI.UI.Controls is deprecated, that's why I can't see it on nuget. So what is the status of this project - I assume this is for things like UniformGrid etc. The project looks alive to me, but all the documentation I find about it leads to non-existing nuget packages. I don't know what's going on. Can I use it? How do I use it if so?

**Comment by Gavin-Williams** [comment]
So, reading this post from last year [Windows Community Toolkit v8.0.230907](https://github.com/CommunityToolkit/Windows/discussions/228) have i got it right that the toolkit is being re-organized. And there's a lot of stuff missing, including a master nuget package for controls? So, in my case, I want to use UniformGrid, if there's no package for it, it's not yet available or won't ever be.

**Comment by Arlodotexe** [comment, marked as answer]
As of 8.0 we no longer have a single 'meta package' for all controls. XAML can't be trimmed, so you'll want to install just the components that you use.

`UniformGrid` is available in the [Primitives](https://github.com/CommunityToolkit/Windows/tree/main/components/Primitives/src) component. For a better overview of the components available in the 8.x version of the community toolkit, we recommend installing our [gallery app](https://apps.microsoft.com/detail/9nblggh4tlcq) (native) or checking out https://toolkitlabs.dev/ (web/Uno).

**Reply by Gavin-Williams** [reply]
Thanks for those links - and "XAML can't be trimmed" - makes sense of the separate packages.
